### DISCUSSION TITLE
[Projects Beta Suggestion] Add a description/notes field so issue/pr descriptions aren’t blank

### DISCUSSION THREAD

**Discussion by Kale-Ko** [community/community#7426] [Projects and Issues] [answered]
I think that it would be really nice if there was a field for setting issue/pr descriptions from the project view so you can add more data to an entry and when viewing an issue/pr in a repo there is a bit more information that you don’t have to just shove it in the title

**Comment by KlausEverWalkingDev** [comment]
But they already exist as a big textarea below the title, aren't they?

**Reply by Kale-Ko** [reply]
In the issue panel yes, but I mean from the projects page

**Comment by henry2man** [comment]
Having description in draft issues might be quite useful.

In my use case, the product owner acts as a non-technical role (he doesn't know about "issues" in the strict meaning of the word), so I've told him that product backlog is like an spreadsheet where he is able to write requests & describe them.

In this case, if he is able to fill some additional info in the Description field this could be useful when the draft issue is converted to a real one, copying the contents of the Description field into the issue.

**Comment by bottlenecked** [comment]
In the old project boards the Notes would also allow for separating text with new lines - this is not supported in the Draft cards which limits note-taking by quite a lot

**Comment by thclark** [comment]
Yeah, I'm missing the old style notes - a single line for the title isn't enough space to capture all the information you require at draft stage.

**Comment by Kale-Ko** [comment, marked as answer]
This has been somewhat implemented now, although it would be nice if the description was still editable after converting to an issue.
